src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -97,11 +101,18 @@ public UserBook findByName(@PathVariable String name) {
     }
 
     @GetMapping("/birthdayAndName")
```

Comment:
also you can use @GetMapping instead of @GetMapping("/birthdayAndName") in this endpoint